src/messages.js

```diff
@@ -145,20 +145,20 @@ export function MsgWithdrawDelegationReward(
   }
 }
 
-function Coin({ amount, denom }) {
+function Coin ({ amount, denom }) {
   return ({
     amount: String(amount),
     denom
   })
 }
 
 export default {
-  "MsgSend": MsgSend,
-  "MsgDelegate": MsgDelegate,
-  "MsgUndelegate": MsgUndelegate,
-  "MsgRedelegate": MsgRedelegate,
-  "MsgSubmitProposal": MsgSubmitProposal,
-  "MsgVote": MsgVote,
-  "MsgDeposit": MsgDeposit,
-  "MsgWithdrawDelegationReward": MsgWithdrawDelegationReward
-}
\ No newline at end of file
+  'MsgSend': MsgSend,
```

Comment:
i think our standard is backticks, not single quotes

Reply:
backticks don't work in object property keys.